db/dropDB.js

```diff
@@ -1,13 +1,8 @@
 const initOptions = {}
 const pgp = require('pg-promise')(initOptions);
-const cn = {
-    host: 'localhost',
-    port: 5432,
-    database: 'postgres',
-    user: 'postgres',
-    password: 'postgres'
-};
-const db = pgp(cn);
+const config = require('./config.json');
+
+const db = pgp(config.db);
```

Comment:
Cannot use same config cuz config connects to fds db. But cannot drop db when u r connected to it